applications/padcirc-frontera/app.json

```diff
@@ -0,0 +1,92 @@
+{
+    "id": "padcirc",
+    "version": "55.01",
+    "description": "PADCIRC is the parallel version of the ADCIRC which is optimized for enhanced performance on multiple computer nodes to run very large models. It includes MPI library calls to allow it to operate at high efficiency on parallel machines.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/padcirc/v55/padcirc.zip",
```

Comment:
I get `permission denied` for this file when I try to run the app published in dev.cep